js/common.js

```diff
@@ -162,12 +163,13 @@ module.exports = {
     if (queryParams.image) {
       const img = queryParams.image;
       delete queryParams.image;
-      // Main magic with files here
+      let resizedImage;
       if (img instanceof Blob) {
-        postData.append('image', img);
+        resizedImage = await resizeImage(img);
```

Comment:
we can set the default to 512, but we should allow them to override the image size as well.
can we add a param to accept the height & width similar to this?
https://github.com/visenze/visearch-sdk-android/blob/master/doc/ViSearch.md#322-resizing-settings